packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -851,41 +882,56 @@ export class BlobManager {
 		for (const [localId, detachedStorageId] of redirectTableEntries) {
 			const newStorageId = detachedStorageTable.get(detachedStorageId);
 			assert(newStorageId !== undefined, 0xc53 /* Couldn't find a matching storage ID */);
-			this.setRedirection(localId, newStorageId);
+			this.redirectTable.set(localId, newStorageId);
 			// set identity (id -> id) entry
-			this.setRedirection(newStorageId, newStorageId);
+			this.redirectTable.set(newStorageId, newStorageId);
 		}
 	};
 
+	/**
+	 * Upload and attach any pending blobs that the BlobManager was loaded with that have not already
+	 * been attached in the meantime.
+	 * @returns A promise that resolves when all the uploads and attaches have completed, or rejects
+	 * if any of them fail.
+	 */
+	public readonly sharePendingBlobs = async (): Promise<void> => {
```

Comment:
this is not being used anywhere?

Reply:
Correct - this is new functionality that doesn't yet have an entrypoint.  It should be conditionally called only for non-frozen containers (not sure if we have a way to distinguish these from frozen containers currently, but that will be required before we can call it safely).  There's one test that calls it, and I'll be adding more tests for it in my next change.